### PR TITLE
[Rule Tuning] Feedback from 7.12 Kibana PR

### PR DESCRIPTION
**Pull request by brokensound77**
## Summary
Tuning and modifications from feedback related to the rules [update](https://github.com/elastic/kibana/pull/91082) to Kibana

### CODE REVIEW

**Review by bm11100 — COMMENTED** [review]
Can we also add the latter path below to `Suspicious JAR Child Process` if we are keeping that [rule](https://github.com/elastic/kibana/pull/91082#discussion_r574542204) in - 
`not process.executable : ("/Users/*/.sdkman/*", "/Library/Java/JavaVirtualMachines/*")`

**Comment by bm11100** [comment]
Also, need to correct the misspelling `expoit` in the FP section of `Sudo Heap-Based Buffer Overflow Attempt`

**Review by bm11100 — APPROVED** [review]
Two comments, otherwise LGTM

File: rules/windows/discovery_file_dir_discovery.toml [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-updated_date = "2020/01/28"
+updated_date = "2020/02/11"
 
 [rule]
 author = ["Elastic"]
 description = """
 Enumeration of files and directories using built-in tools. Adversaries may use the information discovered to plan
 follow-on activity.
 """
+false_positives = [
```
Comment (brokensound77) [inline rules/windows/discovery_file_dir_discovery.toml:12]: awesome

**Comment by brokensound77** [comment]
Thanks @bm11100, changes LGTM

File: rules/windows/discovery_file_dir_discovery.toml [hunk 2]
```diff
@@ -20,9 +27,16 @@ tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
 type = "eql"
 
 query = '''
-process where event.type in ("start", "process_started") and
-  (process.name : "cmd.exe" or process.pe.original_file_name == "Cmd.Exe") and
-  process.args : ("dir", "tree")
+sequence by process.entity_id with maxspan=1m
```
Comment (rw-access) [inline rules/windows/discovery_file_dir_discovery.toml:30]: smart!

but these wont all have the same `process.entity_id`. maybe this?
```suggestion
sequence by agent.id, user.name with maxspan=1m
```

**Review by rw-access — APPROVED** [review]
one correction, otherwise this is good.
you da real mvp

**Pull request merged**